apps/chat/src/components/Promptbar/components/PromptbarSettings.tsx

```diff
@@ -50,6 +50,7 @@ export function PromptbarSettings() {
         Icon: IconSquareCheck,
         onClick: () => {
           dispatch(PromptsActions.setAllChosenPrompts());
+          dispatch(PromptsActions.setAllChosenEmptyFolders());
```

Comment:
Why do we need separate action for that? We can do it into `setAllChosenPrompts`

Reply:
I think it's better to implement separate actions for improved separation of concerns, testability, and reusability.

Reply:
Code changed as suggested.